finagle/h2/src/main/scala/com/twitter/finagle/buoyant/h2/Stream.scala

```diff
@@ -162,24 +186,14 @@ object Stream {
   def const(s: String): Stream =
     const(Buf.Utf8(s))
 
-  def empty(): Stream with Writer = empty(new AsyncQueue[Frame](1))
-
-  def empty(frameQ: AsyncQueue[Frame]): Stream with Writer =
```

Comment:
An empty Stream should never have frames written to it or read from it.  The only reason for this queue was to allow cancellation by polling the queue and then immediately interrupting it.  We have a better mechanism for cancellation now (the cancel method).